StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -110,6 +110,7 @@ extension SavedPaymentMethodCollectionView {
         }
 
         var cbcEligible: Bool = false
+        var paymentMethodRemove: Bool = true
```

Comment:
I think this should be called something like `allowsPaymentMethodRemoval` to follow ios naming conventions.

Reply:
That's fine, i was naming it after the name we are using it in customerSession.